VimeoNetworking/Sources/Models/VIMConnection.m

```diff
@@ -90,4 +91,11 @@ - (BOOL)canPost
     return (self.options && [self.options containsObject:@"POST"]);
 }
 
+- (void)setNotifications:(nonnull NSDictionary *)dic
```

Comment:
The issue with having the same as Nicole did is we can't have a `private property` prefixed by `new_`.